react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -428,6 +450,10 @@ class Thumbnail extends Component<Props, State> {
         }
     }
 
+    handleTrackStreamingStatusChanged(jitsiTrack, streamingStatus) {
+        APP.store.dispatch(trackStreamingStatusChanged(jitsiTrack, streamingStatus));
```

Comment:
instead of using `APP.store.dispatch`, will it also work if we add this function to `_mapDispatchToProps` or `this.props.dispatch`?

Reply:
changed